third_party/python.bzl

```diff
@@ -47,11 +47,10 @@ def tensorboard_python_workspace():
     http_archive(
         name = "org_html5lib",
         urls = [
-            "http://mirror.tensorflow.org/github.com/html5lib/html5lib-python/archive/1.0.1.tar.gz",
```

Comment:
Let's keep a mirror URL. We can add one for this if it doesn't exist yet (go/tensorboard-tf-mirror), but it looks like there is one for http://mirror.tensorflow.org/github.com/html5lib/html5lib-python/archive/1.1.tar.gz already so that's probably easiest.

Reply:
Makes sense. Updated to the familiar mirrors and calculated the checksum with `sha256sum`.